frontend/next.config.js

```diff
@@ -30,6 +30,12 @@ const nextConfig = {
       },
     ];
   },
+  i18n: {
+    locales: ['default', 'en', 'fr', 'es', 'zh-CN', 'pt-BR', 'ru'],
+    defaultLocale: 'default',
```

Comment:
Can you explain what is this `default` locale? Why are we not defaulting to `en`?

Reply:
This is just a trick on the Next documentation to add the 'en' prefix if we don't have any specified https://nextjs.org/docs/advanced-features/i18n-routing#prefixing-the-default-locale

Reply:
Ok, got it. Do we need to prefix the default locale though? If we don't, you don't have to update all the Cypress tests.

I think it's ok to go to the English version by visiting resilienceatlas.org and not resilienceatlas.org/en.